core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -218,10 +232,20 @@ public MultiMatchQueryBuilder slop(int slop) {
     }
 
     /**
+     * Set the phrase slop if evaluated to a phrase query type.
+     */
+    public MultiMatchQueryBuilder slop(Integer slop) {
+        this.slop = slop;
```

Comment:
I agree we should actually set defaults here, and find these defaults in the actual MatchQuery.